Jamulus.pro

```diff
@@ -93,22 +93,31 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+        	message("Native i386 build")
```

Comment:
Is this correct?  What if I'm running Windows 64-bit but a 32bit compile?  That's _not_ a native i386 build.

Reply:
I would say If you are running a Windows 64 bits, QT_ARCH will be set to `x86_64`.